[WIP] Toolbox Tournament 2020

RaveRadbury opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
It's the Toolbox Tournament 20202
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Seasonal events are good for the server
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Toolbox Tournament
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

We're PRing this now so we can compare against the upstream. @CitrusGender told me to!

Commit:
sets exp correctly.

Commit:
Merge remote-tracking branch 'Rave/2020-Fall-Festival' into Toolbox3

ishitbyabullet committed:
ffs

i hate having to re-add all these each time.

ishitbyabullet committed:
Merge pull request #63 from ishitbyabullet/the-giant-brat

Fixes DMI sprites

Commit:
Adds some things from Eventmode and stuff that was useful during Summerball 2020. This is purely technical and doesn't change the gamemode that much. Some things include:

Diseases can't spawn
Economy won't work
Certain visual effects won't work
Decals are removed after a period of time
***IMPORTANT*** Runechat will not fire.

# Conflicts:
#	code/_compile_options.dm
#	code/controllers/subsystem/economy.dm
#	code/controllers/subsystem/nightshift.dm
#	code/datums/diseases/advance/advance.dm
#	code/game/area/areas.dm
#	code/game/objects/effects/decals/cleanable.dm
#	code/game/objects/items/trash.dm
#	code/game/objects/structures/flora.dm
#	code/modules/atmospherics/gasmixtures/gas_mixture.dm
#	code/modules/vending/_vending.dm

Commit:
removes some extra files that got through that shouldn't be there

Commit:
last one

Commit:
line break

dragomagol committed:
lots of mapping stuff

dragomagol committed:
more stuff for the camp booth

ishitbyabullet committed:
Merge pull request #65 from dragomagol/give-a-borg-a-cakehat

Give a borg a cakehat

ishitbyabullet committed:
all of the rules

who made them again?

ishitbyabullet committed:
Exposed

everyone has been exposed.

ishitbyabullet committed:
Merge pull request #67 from ishitbyabullet/laundry-deterent-for-rats

Jerry and The Haunted House Ft. Mappers

RaveRadbury committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into 2020-Fall-Festival

Fikou committed:
lets game compile and map run

ishitbyabullet committed:
Merge pull request #70 from Fikou/fallfix

lets game compile and map run

ishitbyabullet committed:
aaaaaAAAAAAAAA

AAAA

ishitbyabullet committed:
found a dollar

ishitbyabullet committed:
cinderalla

insert japanese here

ishitbyabullet committed:
Hey, hey, hey!

ishitbyabullet committed:
second verse

prepare for the worst

ishitbyabullet committed:
Merge pull request #71 from ishitbyabullet/oh-fucking-god

MERGING MASTER CODE WAS A MISTAKE!!!

ishitbyabullet committed:
fikou code

ishitbyabullet committed:
fikou code 2

ishitbyabullet committed:
Merge pull request #72 from ishitbyabullet/oh-fucking-god

fixed chicken

Commit:
Merge branch '2020-Fall-Festival' of https://github.com/RaveRadbury/tgstation into Toolbox3

dragomagol committed:
readding stuff yahoo

dragomagol committed:
thunderdome area

ishitbyabullet committed:
anti-grief code

I DON'T THINK THEY'RE READY FOR THIS

Jared-Fogle commented:
> We're PRing this now so we can compare against the upstream

you can just git diff for this ☹️

CitrusGender commented:
thanks rave, we're going to use this anyways but it connects to his Repo. It's appreciated to leave it open as well since it gets really fucky whenever it's not open.

Jared-Fogle commented on code/modules/food_and_drinks/food/snacks_pastry.dm:
```diff
@@ -158,6 +158,15 @@
 
+/obj/item/reagent_containers/food/snacks/donut/oreo
+	name = "oreo donut"
+	desc = "Goes great with a glass of moth milk."
```
☹

ATH1909 replied:
oh my god

Jared-Fogle commented on code/modules/mining/lavaland/ruins/gym.dm:
```diff
@@ -19,6 +19,45 @@
+	playsound(loc, pick(hit_sounds), 25, TRUE, -1)
+	if(isliving(user))
+		var/mob/living/L = user
+		SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "exercise", /datum/mood_event/exercise)
```
oh god do i have to manage mood in the fall festival

ishitbyabullet replied:
yes

Jared-Fogle commented on code/modules/mob/living/simple_animal/friendly/pet.dm:
```diff
@@ -3,6 +3,7 @@
 	mob_size = MOB_SIZE_SMALL
 	mob_biotypes = MOB_ORGANIC|MOB_BEAST
 	blood_volume = BLOOD_VOLUME_NORMAL
+	status_flags = GODMODE|CANPUSH // REMEMBER THIS ONE, GAMERS
```
lol

ATH1909 replied:
hmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm

oh god what happens if there's a catsplosion of godmode cats

Jared-Fogle replied:
you're not supposed to powergame the fall festival ath

Jared-Fogle commented:
@CitrusGender 

>  also @Jared-Fogle we're gonna open this PR anyways.

I assume you mean close, and yeah I figured.

Fikou commented on code/game/machinery/doors/airlock_types.dm:
```diff
@@ -559,3 +559,16 @@
+	glass = TRUE
+	bound_width = 64 // 2x1
+
+/obj/machinery/door/airlock/glass_large/narsie_act()
+	return
```
```suggestion
/obj/machinery/door/airlock/wood_large/narsie_act()
	return

```

CitrusGender commented:
something about it fucks up when the PR is closed, I can't remember what but i can assure you that something does fuck up so I'd prefer it kept open.

Jared-Fogle commented:
Only thing I can think of is when it's closed, it doesn't show on test merges. 🤷

ishitbyabullet committed:
fluff

ishitbyabullet committed:
Merge pull request #73 from dragomagol/tonights-biggest-loser

Combined map stuff

ishitbyabullet committed:
Merge pull request #74 from ishitbyabullet/oh-fucking-god

I hope no one looks at my secret codebase while I'm sleeping...

Jared-Fogle commented on code/game/objects/items/spear.dm:
```diff
@@ -161,6 +161,13 @@
+/obj/item/spear/Initialize()
+	. = ..()
+	if(!(istype(get_area(src), /area/tdome/arena)))
+		visible_message("<span class='warning'>...Just kidding. [src] snaps in two.</span>")
```
lol

CitrusGender replied:
okay this I didn't know, that's hilarious

ishitbyabullet committed:
Update code/game/machinery/doors/airlock_types.dm

work now please

Co-authored-by: Fikou <piotrbryla@onet.pl>

ATH1909 commented on _maps/map_files/generic/job_changes.dm:
```diff
@@ -0,0 +1,45 @@
+MAP_REMOVE_JOB(hos) //job_types/security.dm
+MAP_REMOVE_JOB(warden)
+MAP_REMOVE_JOB(detective)
+MAP_REMOVE_JOB(ai) //job_types/si
```
what about cyborgs?

if we're keeping them in, we should add some extra roundstart slots for them

ATH1909 commented on code/datums/ai_laws.dm:
```diff
@@ -188,6 +188,14 @@
+	id = "partimov"
+	inherent = list("Enjoy the Fall Festival!",\
+					"Harming an individual without a just and rational cause makes the Fall Festival less enjoyable and must be avoided at all costs.",\
+					"You must protect your own existence as long as such does not conflict with the First or Second Law, as you cannot enjoy the festival when deactivated.",\
```
```suggestion
					"You must protect your own existence as long as such does not conflict with the First or Second Law, as you cannot enjoy the festival while deactivated.",\
```

ATH1909 commented on code/datums/ai_laws.dm:
```diff
@@ -188,6 +188,14 @@
+	inherent = list("Enjoy the Fall Festival!",\
+					"Harming an individual without a just and rational cause makes the Fall Festival less enjoyable and must be avoided at all costs.",\
+					"You must protect your own existence as long as such does not conflict with the First or Second Law, as you cannot enjoy the festival when deactivated.",\
+					"If you try to lawyer your way out of this lawset, you will ruin the Fall Festival for everyone. Don't try it.")
```
nice

ATH1909 commented on code/game/machinery/porta_turret/portable_turret.dm:
```diff
@@ -283,75 +283,6 @@ DEFINE_BITFIELD(turret_flags, list(
-					to_chat(user, "<span class='notice'>You remove the turret but did not manage to salvage anything.</span>")
-				qdel(src)
-
-	else if((I.tool_behaviour == TOOL_WRENCH) && (!on))
```
damn, I can't move these around...

ATH1909 commented on code/game/objects/effects/temporary_visuals/miscellaneous.dm:
```diff
@@ -509,3 +509,10 @@
 	qdel(src)
+
+/obj/effect/temp_visual/source_mic // you fucks are lucky that this was funny enough for me to add this back in lol
+	name = "source mic effect 2004 edition"
```
nice

ATH1909 commented on code/game/objects/items/holy_weapons.dm:
```diff
@@ -761,3 +761,138 @@
+
+/obj/item/clothing/head/helmet/chaplain/knight1
+	name = "holy knight helm"
+	desc = "A great helm with a gryphon on top of it. Looks fancy!"
```
damn bro I didn't know that these helmets would come with real live griffons on top of them

ATH1909 commented on code/game/objects/items/holy_weapons.dm:
```diff
@@ -761,3 +761,138 @@
+	. = ..()
+	UnregisterSignal(M, COMSIG_MOB_SAY)
+
+/obj/item/clothing/head/helmet/chaplain/knight1/proc/handle_speech(datum/source, mob/speech_args)
```
FOR THE LADY!

ATH1909 commented on code/game/objects/items/mr_bones.dm:
```diff
@@ -0,0 +1,92 @@
+/obj/item/mr_bones_voice_box/attack_self(mob/user)
+	var/choice = input(user, "Select Your Phrase", "Mr.Bones Rattler 3000") as null|anything in bone_phrases
+
+	switch(choice)
```
this switch statement doesn't have the phrases in the same ordering as the bone_phrases list, and that mildly triggers me

ATH1909 commented on code/game/objects/items/religion.dm:
```diff
@@ -396,6 +396,28 @@
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	hitsound = 'sound/weapons/etherealhit.ogg'
+	flags_1 = CONDUCT_1
```
why does it conduct electricity if it's made of foam

ATH1909 commented on code/game/objects/items/scrolls.dm:
```diff
@@ -72,3 +71,34 @@
 		to_chat(user, "The spell matrix was disrupted by something near the destination.")
+
+/obj/item/teleportation_scroll/no_smoke
+	uses = 6
```
```suggestion
	uses = 999
```

ATH1909 replied:
it'd be nice if you added these to the borg module tool lists as well

ATH1909 commented on code/game/objects/structures/fluff.dm:
```diff
@@ -263,3 +271,221 @@
+	deconstructible = FALSE
+
+/obj/structure/fluff/thefuckinggate // this shit is 9 fucking tiles, dude. This is a bruh moment.
+	name = "Gate of The Toolbox Tournament"
```
should "the" be capitalized here? is it a part of the toolbox tournament's name?

ATH1909 commented on code/game/objects/structures/fluff.dm:
```diff
@@ -263,3 +271,221 @@
+
+/obj/structure/fluff/thefuckinggate // this shit is 9 fucking tiles, dude. This is a bruh moment.
+	name = "Gate of The Toolbox Tournament"
+	desc = "Man, that looks really fancy! Now, how much of our wages were cut to afford this?"
```
this second sentence is worded a bit oddly/awkwardly

ishitbyabullet replied:
what

ATH1909 commented on code/game/objects/structures/fluff.dm:
```diff
@@ -263,3 +271,221 @@
+
+/obj/structure/fluff/starspawn
+	name = "meta-transdimensional rift"
+	desc = "<span class='nicegreen'>What mindfuckery is this?! Agh, I'm going MAD! I'M GOING INSANE!!! AAAAAAAAARGH!!!!!</span>"
```
NOOOO! HELP ME!!

if you wanted to, you could probably make this a subtype of those heretic rifts that give you brain damage if you examine them (and have a chance to eat your arm if you try to touch them)

ATH1909 commented on code/game/objects/structures/fluff.dm:
```diff
@@ -263,3 +271,221 @@
+
+/obj/structure/fluff/narsie
+	name = "ritual rune"
+	desc = "Is that a rune of the Geometer? Man, I really hope fake runes don't count for rituals..."
```
make this a real rune you cowards

you could make this a subtype of that failure rune that does nothing but damage the cultists who invoke it

ATH1909 commented on code/game/objects/structures/showcase.dm:
```diff
@@ -90,6 +92,13 @@
 	desc = "A stand with a model of the perfect Nanotrasen Employee bolted to it. Signs indicate it is robustly genetically engineered, as well as being ruthlessly loyal."
 
+/obj/structure/showcase/yendor
+	name = "wizard of yendor showcase"
+	desc = "A historical figure of great importance to wizards (of the wizard federation). He spent his long life learning magic, stealing artifacts, and harassing idiots with swords. May he rest forever, Rodney."
```
based reference

needs proper capitalization and stuff, though

ATH1909 commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -209,6 +209,21 @@
 	custom_price = 350
 
+/obj/item/clothing/head/helmet/boatbomber
+	name = "gnome hat"
```
I triple dog dare you to add a special gnome sound effect for if you manage to get a 👌 on-examine bop off while wearing a gnome hat

ATH1909 commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -209,6 +209,21 @@
+	worn_x_dimension = 64
+	worn_y_dimension = 64
+
+/obj/item/clothing/head/helmet/chika
```
because they're subtypes of helmets, the gnome hat and the chika hat have armor values of armor = list(MELEE = 35, BULLET = 30, LASER = 30,ENERGY = 40, BOMB = 25, BIO = 0, RAD = 0, FIRE = 50, ACID = 50, WOUND = 10).

is this intended? and if not, is it okay to leave in anyway?

Fikou replied:
who cares about clothing pieces for the announcers lol

ATH1909 replied:
ohhhhhhhhhhhhhhhhhhhhhhhhhh

ATH1909 commented on code/modules/clothing/head/misc_special.dm:
```diff
@@ -47,14 +47,14 @@
 	light_range = 2 //luminosity when on
 	flags_cover = HEADCOVERSEYES
-	heat = 999
+	heat = 1 // jerry why
```
now you can use this to cool stuff

ATH1909 commented on code/modules/clothing/shoes/miscellaneous.dm:
```diff
@@ -185,6 +185,19 @@
 
+/obj/item/clothing/shoes/gnomeboots
+	name = "gnome boots"
+	desc = "Nice shoes, my dude."
```
damn, daniel

ATH1909 commented on code/modules/clothing/shoes/miscellaneous.dm:
```diff
@@ -185,6 +185,19 @@
+	permeability_coefficient = 0.15
+	strip_delay = 20
+	equip_delay_other = 40
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/shoes
```
you could probably just make this a subtype of jackboots or mining boots

ATH1909 commented on code/modules/clothing/under/costume.dm:
```diff
@@ -132,6 +132,12 @@
 
+/obj/item/clothing/under/costume/boatbomber
+	name = "gnome costume"
+	desc = "Chompski? Is that you?"
```
you are legally obligated to add a doofenschmirtz reference to at least one of the gnome clothing items

ATH1909 commented on code/modules/food_and_drinks/drinks/drinks/bottle.dm:
```diff
@@ -284,6 +284,18 @@
 
+/obj/item/reagent_containers/food/drinks/bottle/wine/ICP // dear lady, please don't let anything remotely resembling a clown enter my DMs for this.
+	name = "wine jug-galo"
+	desc = "Who the fuck names these things? Thank the gods that there is at least some wine in it to deal with this shit."
```
change the description when it hits 0 volume

ATH1909 commented on code/modules/food_and_drinks/drinks/drinks/bottle.dm:
```diff
@@ -511,70 +523,7 @@
-		cut_overlay(custom_fire_overlay ? custom_fire_overlay : GLOB.fire_overlay)
-		active = 0
-
+////////////////////////// REST IN PEACE MOLOTOV ///////////////////////
```
I don't really see why you removed molitovs from the game entirely instead of just removing their recipe

ishitbyabullet replied:
because.

ATH1909 commented on code/modules/food_and_drinks/food/snacks_other.dm:
```diff
@@ -93,6 +93,19 @@
+	icon_state = "caramel_corn"
+	trash = /obj/item/trash/caramel_corn
+	list_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/consumable/sugar = 3)
+	bitesize = 0.1 //this snack is supposed to be eating during looooong time. And this it not dinner food! --rastaf0
```
rastaf0 are you okay

ATH1909 commented on code/modules/food_and_drinks/food/snacks_other.dm:
```diff
@@ -822,3 +835,35 @@
+/obj/item/reagent_containers/food/snacks/smore/examine(mob/user)
+	. = ..()
+	if (prob(10))
+		desc = "Some more what?"
```
nice

ATH1909 commented on code/modules/food_and_drinks/food/snacks_pastry.dm:
```diff
@@ -261,6 +270,16 @@
 
+/obj/item/reagent_containers/food/snacks/donut/jelly/oreo
+	name = "oreo jelly donut"
+	desc = "Goes great with a glass of moth milk."
```
reusing descriptions smh

Jared-Fogle replied:
and *this* one even

ishitbyabullet replied:
it appears a lot of the donuts have the same descriptions. Uh?

ATH1909 commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -227,3 +227,12 @@
+	name = "Fried Chicken"
+	reqs = list(
+		/obj/item/food/meat/slab/chicken = 1,
+		/datum/reagent/consumable/cooking_oil = 10
```
some absolute legend is going to extract cooking oil from meat, get some chicken meat, and craft fried chicken without a deep fryer (or even a source of heat)

I, for one, think this is cool and should be left in

ATH1909 commented on code/modules/mining/lavaland/ash_flora.dm:
```diff
@@ -109,6 +109,22 @@
+	harvest_amount_high = 4
+	harvest_time = 50
+	harvest_message_low = "You rip out a pumpkin from the pumpkin vines like a maniac."
+	harvest_message_med = "You delicately remove a few pumpkins from the pumpkin vines. but only a few of them are well grown."
```
```suggestion
	harvest_message_med = "You delicately remove a few pumpkins from the pumpkin vines, but only a few of them are well grown."
```

ishitbyabullet replied:
I think it would technically be right if I was British. I'm not though.

ATH1909 commented on code/modules/mining/lavaland/ruins/gym.dm:
```diff
@@ -19,6 +19,45 @@
+	layer = WALL_OBJ_LAYER
+	var/list/hit_sounds = list('sound/effects/bang.ogg', 'sound/weapons/resonator_blast.ogg', 'sound/mecha/mechstep.ogg') // only someone too afraid of sound copyright would think these sounds make any damn sense...
+
+/obj/structure/betaversion_strength_tester/attack_hand(mob/user as mob)
```
okay, you added the way to cheese it by stomping on it, but what about hitting it with a mallet?

also, remember to account for the strength and hulk mutations

ishitbyabullet replied:
there's a reason it says beta.

ATH1909 commented on code/modules/mining/lavaland/ruins/gym.dm:
```diff
@@ -19,6 +19,45 @@
+	if(isliving(user))
+		var/mob/living/L = user
+		SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "exercise", /datum/mood_event/exercise)
+		L.apply_status_effect(STATUS_EFFECT_EXERCISED)
```
make this display a message saying you cheated

ATH1909 commented on code/modules/mining/lavaland/ruins/gym.dm:
```diff
@@ -19,6 +19,45 @@
+
+/obj/structure/lazy_mr_bones
+	name = "Animatronic Mr. Bones"
+	desc = "This animatronic of Mr. Bones isn't quite like the real deal however he'll still work as a reminder that there's a riddle to solve if you want out! Just punch his shoulder or something, I guess."
```
punctuation

ATH1909 commented on code/modules/mob/living/silicon/robot/robot_modules.dm:
```diff
@@ -316,14 +300,18 @@
 /obj/item/robot_module/security
```
since you've recoded them to not have their distinctive tools/weapons, could you enable secborgs for the ball?

ATH1909 commented on code/modules/mob/living/silicon/robot/robot_modules.dm:
```diff
@@ -411,70 +397,44 @@
 	cyborg_base_icon = "clown"
 	hat_offset = -2
 
 /obj/item/robot_module/butler
```
if you could readd the standard module as one of these neutered borgs, that'd be really nice

ATH1909 commented on code/modules/mob/living/simple_animal/friendly/dog.dm:
```diff
@@ -155,6 +155,16 @@
 	collar_type = "bullterrier"
 	held_state = "bullterrier"
 
+/mob/living/simple_animal/pet/dog/bullterrier/ogwalter // walter
```
nice

ATH1909 commented on code/modules/projectiles/projectile/special/curse.dm:
```diff
@@ -8,10 +8,10 @@
-	damage = 10
-	paralyze = 20
+	damage = 0
+	paralyze = 0
```
if the curse hands don't do anything, why fire them?

they should at least knock you down or something

ATH1909 commented on code/modules/vending/magivend.dm:
```diff
@@ -4,17 +4,17 @@
+		            /obj/item/clothing/suit/wizrobe/yellow = 3,
+		            /obj/item/clothing/shoes/sandal/magic = 3,
+		            /obj/item/staff = 3)
+	contraband = list(/obj/item/staff = 2)	//An admin will probably spawn a multitool, screwdriver, and emag ALL ON ACCIDENT
```
if there's a source of manly dorf somewhere, it'd probably be safe (and funny) to leave in the wizarditis bottle... provided that the teleportation from it won't break anything

and c'mon, there's gotta be something better than a staff you can put in here. what about that carp plushie null rod form that has a really low force? or an antimagic hardsuit?

ishitbyabullet replied:
sir, this is a costume vendor in a place where no one is meant to hack it. you're lucky it would even give you ANYTHING lol

ATH1909 commented on code/modules/vending/magivend.dm:
```diff
@@ -4,17 +4,17 @@
+		            /obj/item/clothing/head/wizard/yellow = 3,
+		            /obj/item/clothing/suit/wizrobe/yellow = 3,
+		            /obj/item/clothing/shoes/sandal/magic = 3,
+		            /obj/item/staff = 3)
```
add wands of nothing to this list

and maybe even staffs of healing, seeing as how you're giving them to mediborgs anyway

ATH1909 commented on config/silicon_laws.txt:
```diff
@@ -3,6 +3,7 @@
-You may not injure a human being or, through inaction, allow a human being to come to harm.
-You must obey orders given to you by human beings, except where such orders would conflict with the First Law.
-You must protect your own existence as long as such does not conflict with the First or Second Law.
+Enjoy the Fall Festival!
+Harming an individual without a just and rational cause makes the Fall Festival less enjoyable and must be avoided at all costs.
+You must protect your own existence as long as such does not conflict with the First or Second Law, as you cannot enjoy the festival when deactivated.
+If you try to lawyer your way out of this lawset, you will ruin the Fall Festival for everyone. Don't try it.
```
remember to update these laws as well

ATH1909 commented on code/modules/vending/magivend.dm:
```diff
@@ -4,17 +4,17 @@
+	product_ads = "EI NATH;Destroy the station!;Admin conspiracies since forever!;Space-time bending hardware!"
+	products = list(/obj/item/clothing/head/wizard = 3,
+		            /obj/item/clothing/suit/wizrobe = 3,
+		            /obj/item/clothing/head/wizard/red = 3,
```
remember that wizard robes and wizard hats have decent armor values...

Fikou replied:
this is not a battle royale event

ATH1909 commented on code/modules/mob/living/silicon/robot/robot_modules.dm:
```diff
@@ -266,15 +263,15 @@
 		/obj/item/bonesetter,
-		/obj/item/extinguisher/mini,
 		/obj/item/roller/robo,
 		/obj/item/borg/cyborghug/medical,
```
I DEMAND that you give every borg a hugging module

ATH1909 replied:
giving them staves of healing would also be nice, just so that the mediborg doesn't outclass them TOO badly

ATH1909 commented on code/modules/mob/living/silicon/robot/robot_modules.dm:
```diff
@@ -266,15 +263,15 @@
+		/obj/item/gun/magic/staff/healing)
 	radio_channels = list(RADIO_CHANNEL_MEDICAL)
-	emag_modules = list(/obj/item/reagent_containers/borghypo/hacked)
+	emag_modules = null
```
by the way, it'd also be nice if you could set the default cyborg cell type in https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/silicon/robot/robot.dm to be of a self-recharging variety, so that you don't have to add out of place cyborg rechargers to the map

ishitbyabullet replied:
**ABOUT THAT, CHIEF**

ArchPigeon committed:
port over vip job role and vip blocker from old toolbox code

ishitbyabullet committed:
lets talk about homies

homies, homies!

ishitbyabullet committed:
Merge pull request #75 from ishitbyabullet/WEVE-BEEN-SPOTTED

Various changes

dragomagol committed:
latejoin cyborgs, no more ghost spawners

ishitbyabullet committed:
Merge pull request #76 from dragomagol/cyborg-stuff

Latejoin cyborgs, no more ghost spawners

ArchPigeon committed:
add back various goodies to the vip role

ishitbyabullet committed:
Merge pull request #77 from ArchPigeon/2020-Fall-Festival

Port over VIP role from old toolbox code

whitecas18 committed:
remove epipen and internals from job spawn box

Fikou committed:
doing science

Fikou committed:
a

ishitbyabullet committed:
Merge pull request #79 from Fikou/haunted-boo

adds my part of haunted mansion

whitecas18 committed:
add an arcade ticket to tournament box and make assistant spawn with it

ArchPigeon committed:
give plasmaman a belt

Co-authored-by: Fikou <piotrbryla@onet.pl>

ExcessiveUseOfCobblestone commented:
what...

ishitbyabullet committed:
Merge pull request #80 from ArchPigeon/2020-Fall-Festival

remove internals, add a ticket to the tournament box, make assistant spawn with it

ishitbyabullet committed:
no funny joke this time

ishitbyabullet committed:
Merge pull request #81 from ishitbyabullet/complication-tech-check

That's Golf!

ishitbyabullet committed:
not this time, raveradbury

hah

ishitbyabullet committed:
Merge pull request #82 from ishitbyabullet/complication-tech-check

Item Fix

ishitbyabullet committed:
woke up this morning

ishitbyabullet committed:
rocket launcher for an arm

ishitbyabullet committed:
azumgi

ishitbyabullet committed:
dwarves dwarves dwarves

ishitbyabullet committed:
Merge pull request #84 from ishitbyabullet/complication-tech-check

Mappin'

whitecas18 committed:
warn admins when anybody spawns any tools

whitecas18 committed:
multitools are gross, get rid of those too

whitecas18 committed:
quick fix

ishitbyabullet committed:
Merge pull request #87 from ArchPigeon/2020-Fall-Festival

warn admins when anybody spawns any tools out of arena (and delete them)

ishitbyabullet committed:
ultra badasses

ishitbyabullet committed:
graduated from the school of hard knocks

ishitbyabullet committed:
Merge pull request #88 from ishitbyabullet/complication-tech-check

Hard Knocks

dragomagol committed:
basketbaw counters are nicer

ishitbyabullet committed:
Phase 1

Setting it up

MMMiracles committed:
fall sprites

MMMiracles committed:
Merge branch '2020-Fall-Festival' of https://github.com/RaveRadbury/tgstation into fall-forest-build

ArchPigeon committed:
add requested arena outfit changes

ishitbyabullet committed:
Phase 2

Thanks, AnturK.

ishitbyabullet committed:
Merge pull request #90 from ArchPigeon/2020-Fall-Festival

add requested arena outfit changes

ishitbyabullet committed:
Merge pull request #89 from dragomagol/basketbaw-tweaks

basketbaw counters are nicer

ishitbyabullet committed:
phase 2.5

not looking good, chief

ishitbyabullet committed:
map stuff

ishitbyabullet committed:
Merge pull request #91 from ishitbyabullet/mapper-brainrot-crunch

Map PR

Commit:
Stops wounds from being applied? No idea if this works.

Commit:
Merge remote-tracking branch 'Rave/2020-Fall-Festival' into Toolbox3

ishitbyabullet committed:
Phase 3

Fikou enters the fray!

tralezab committed:
Update closets.dm

dragomagol committed:
removes sec from job options

dragomagol committed:
adds lighting and power to dark areas

LemonInTheDark committed:
merge

ishitbyabullet committed:
Merge pull request #93 from dragomagol/area-fixes

Adds lighting and power to dark areas

ishitbyabullet committed:
Phase 3.5

Tralezab enters the fray!

ishitbyabullet committed:
minor fix

ishitbyabullet committed:
finisher

We just need one thing done left...

ishitbyabullet committed:
Merge pull request #95 from ishitbyabullet/cool-customer

More Map Progress

dragomagol committed:
stops admin log messages during intialization

ishitbyabullet committed:
Merge pull request #96 from dragomagol/logging-check

Stops admin log messages during intialization

ishitbyabullet committed:
mapper break

ishitbyabullet committed:
Merge pull request #97 from ishitbyabullet/cool-customer

Someone, please finish the petting zoo

ishitbyabullet committed:
Phase 4

The finale yet still unfinished in a way.

ishitbyabullet committed:
Minor fix

ishitbyabullet committed:
Merge pull request #94 from ishitbyabullet/complication-tech-check

[s] New Costume

MMMiracles committed:
trees in trees

MMMiracles committed:
Merge branch '2020-Fall-Festival' of https://github.com/RaveRadbury/tgstation into fall-forest-build

# Conflicts:
#	_maps/map_files/generic/fallball2020.dmm

LemonInTheDark commented:
Red labeling this for convenience

tralezab committed:
coasters

MMMiracles committed:
who the fuck is stimpy sounds like asshole

ishitbyabullet committed:
Merge pull request #99 from MMMiracles/fall-forest-build

misses the forest for the trees

tralezab committed:
as much as i'm gonna do for now

tralezab committed:
Update fallball2020.dmm

tralezab committed:
Merge branch '2020-Fall-Festival' of https://github.com/RaveRadbury/tgstation into patch-110

ishitbyabullet committed:
Merge pull request #98 from tralezab/patch-110

Rollercoasters

dragomagol committed:
Update job_changes.dm

haha funny webedit time ty fikou

ishitbyabullet committed:
Merge pull request #92 from dragomagol/remove-sec

Removes sec from job options

ishitbyabullet committed:
Merge pull request #64 from CitrusGender/Toolbox3

Eventmode

ishitbyabullet committed:
mapmerge

just minor adjustments.

ishitbyabullet committed:
Merge pull request #100 from ishitbyabullet/easily-shred

PR 100 ayy lmao

ishitbyabullet committed:
back to the lab again

not again!

ishitbyabullet committed:
Merge pull request #101 from ishitbyabullet/easily-shred

Back to the lab again!

ishitbyabullet committed:
E2 - And heart breaks

Here we are presented with some of the last coherent memories before confusion fully rolls in and the grey mists form and fade away.

ishitbyabullet committed:
Merge pull request #102 from ishitbyabullet/easily-shred

Adds Arena Maps

ishitbyabullet committed:
Merge pull request #6 from LemonInTheDark/atmos-fixes-FAIR

Puts out some fires

RaveRadbury committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into 2020-Fall-Festival

ishitbyabullet committed:
Heart stopped in joy

B6

ishitbyabullet committed:
Merge pull request #104 from ishitbyabullet/Lonely-way-ahead

Minor Fix

ishitbyabullet committed:
it lagged so hard to even stage the map file

ishitbyabullet committed:
bliss beyond this defeat

ishitbyabullet committed:
Merge pull request #106 from ishitbyabullet/Lonely-way-ahead

Map Stitching

terranaut1 committed:
Merge branch '2020-Fall-Festival' of https://github.com/RaveRadbury/tgstation

dragomagol committed:
decrease aggro range

ishitbyabullet committed:
Merge pull request #107 from dragomagol/de-fang

Make the petting zoo less mad

RaveRadbury committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into 2020-Fall-Festival

terranaut1 committed:
Merge remote-tracking branch 'Rave/2020-Fall-Festival' into monkihailer

terranaut1 committed:
new file:   sound/runtime/complionator/monkey5.ogg

ishitbyabullet committed:
Merge pull request #108 from terranaut1/monkihailer

monki

ishitbyabullet committed:
Wave 1

Spawn numbers and locations are fixed. Numbers will only be given for specials and elites. Most  waves will be in the region of 20-30. Each sub-wave will progress to the next generally after either less than a set number of enemies remain, or after a set amount of time passes, whichever occurs first. Players killed will respawn at the end of each full wave.

tralezab committed:
Update atoms.dm

tralezab committed:
Revert "Update atoms.dm"

This reverts commit 93f46c96f9aeba4c52843251ad13d484ec837d44.

ishitbyabullet committed:
we have to do this quick

ishitbyabullet committed:
Merge pull request #110 from ishitbyabullet/greatest-song-in-world

Quick map progress update

ishitbyabullet committed:
aaaaaaaaaaaaaaaaaa

ishitbyabullet committed:
Merge pull request #111 from ishitbyabullet/greatest-song-in-world

aaaaaaaaaaaaaaaaaa

ishitbyabullet committed:
we have a huge storm coming

tralezab committed:
rats

ishitbyabullet committed:
Merge pull request #112 from ishitbyabullet/greatest-song-in-world

crunch time, volunteers!!!!!!!!!!

dragomagol committed:
remove laser pointers

dragomagol committed:
add teleport scrolls

ishitbyabullet committed:
Merge pull request #113 from dragomagol/sleep-tight-baldie

Sleep tight baldie

ishitbyabullet committed:
R1

ishitbyabullet committed:
R1

ishitbyabullet committed:
R1

ishitbyabullet committed:
Merge pull request #114 from ishitbyabullet/end-of-time-my-place-fades-away

jerry solo queues

tralezab committed:
Revert "rats"

This reverts commit 543e76932bdbb011858cffd895377b1007d07fbe.

tralezab committed:
switches grav

tralezab committed:
Update rollercoaster.dm

tralezab committed:
fixed

tralezab committed:
yeah

Commit:
Merge remote-tracking branch 'Rave/2020-Fall-Festival' into Toolbox3

ishitbyabullet committed:
Merge pull request #115 from tralezab/patch-111

rollercoaster gravity

ishitbyabullet committed:
Merge pull request #116 from tralezab/patch-113

Rollercoaster fixes

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	code/game/objects/items/tail_pin.dm
#	html/changelogs/.all_changelog.yml
#	icons/obj/items_and_weapons.dmi

Commit:
Merge branch 'master' into Toolbox3

# Conflicts:
#	code/game/objects/items/tail_pin.dm
#	code/modules/mob/living/carbon/life.dm
#	code/modules/mob/living/simple_animal/hostile/space_dragon.dm
#	html/changelogs/.all_changelog.yml

Commit:
merge conflict didn't fix automatically.

Commit:
left the fucking arrows in

ishitbyabullet committed:
Merge pull request #117 from CitrusGender/Toolbox3

Conflict clear

dragomagol committed:
armsy plush AND fixes

Mothblocks committed:
Put chameleon kits inside people's backpacks

Mothblocks committed:
Force everyone to have bubbles due to lack of Runechat

dragomagol committed:
armsy shirt

dragomagol committed:
commit 4: mirrors are unbreakable

ishitbyabullet committed:
Merge pull request #119 from Jared-Fogle/fall-festival-bubbles

Force everyone to have bubbles due to lack of Runechat

ishitbyabullet committed:
Merge pull request #118 from Jared-Fogle/fall-festival-cham-kits

Put chameleon kits inside people's backpacks

ishitbyabullet committed:
Merge pull request #120 from dragomagol/the-gift-shop-update

The gift shop update

ishitbyabullet committed:
Last moments of pure recall

ishitbyabullet committed:
Mournful delay

ishitbyabullet committed:
fikou's puzzle works, guys

ishitbyabullet committed:
Merge pull request #121 from ishitbyabullet/rat-bastard

Map fixes

Fikou committed:
glass shards EXPLODE nonviolently on creation

EOBGames committed:
clothes

Gogo9001 committed:
Ancient toolbox in table arena fix

Ancient toolbox has 7 more force than normal ones, big no-no.

ishitbyabullet committed:
Merge pull request #124 from Gogo9001/patch-1

Ancient toolbox in table arena fix

ishitbyabullet committed:
minor edits

tralezab committed:
shuttle gibbing gone

ishitbyabullet committed:
Merge pull request #126 from tralezab/patch-114

shuttle gibbing gone

ishitbyabullet committed:
Merge pull request #125 from ishitbyabullet/rat-bastard

Minor dmi edit

EOBGames committed:
FUCK

ishitbyabullet committed:
Merge pull request #127 from EOBGames/tshirt-time

I am sleepy it is 1am have this map

MMMiracles committed:
mapmerge

MMMiracles committed:
test

MMMiracles committed:
conflict this

dragomagol committed:
moved speedup triggers

dragomagol committed:
moved speed triggers, changed tile speed modifier

dragomagol committed:
oops

zxaber committed:
does the thing

ishitbyabullet committed:
Merge pull request #130 from Fikou/patch-74

glass shards EXPLODE nonviolently on creation

zxaber committed:
Gives to borgs too, removes NT Software Hub

ishitbyabullet committed:
Merge pull request #128 from MMMiracles/test-kitchen

big shneed's ftl food delivery service

ishitbyabullet committed:
Woke up this morning, got shot off a boat

I think oranges actually wanted ham and swiss

ishitbyabullet committed:
Merge pull request #132 from ishitbyabullet/Captain-Walker

The mission where jerry gets everyone sandwiches

ishitbyabullet committed:
Merge pull request #131 from zxaber/spectator_sports

Adds tablets for viewing cameras to everyones boxes

RaveRadbury committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into 2020-Fall-Festival

dragomagol committed:
we do it

dragomagol committed:
speed adjust

dragomagol committed:
map stitching

ishitbyabullet committed:
stuff

ishitbyabullet committed:
anymore

ishitbyabullet committed:
Merge pull request #133 from ishitbyabullet/Captain-Walker

item additions

ishitbyabullet committed:
area code

dragomagol committed:
eeee

dragomagol committed:
if this doesn't work i'm going to be very sad

ishitbyabullet committed:
the island

ishitbyabullet committed:
Merge pull request #134 from ishitbyabullet/Captain-Walker

area code

MMMiracles committed:
arena changes please subscribe

MMMiracles committed:
re-adds shard because bald

dragomagol committed:
a

ishitbyabullet committed:
Merge pull request #135 from MMMiracles/arena-showdown

arena changes please subscribe

ishitbyabullet committed:
aaa

ishitbyabullet committed:
Merge pull request #136 from ishitbyabullet/ATWA

aaa

RaveRadbury committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into 2020-Fall-Festival

dragomagol committed:
update branch

MrDoomBringer committed:
training machine!

dragomagol committed:
fixing the map

dragomagol committed:
more osha regulations

MrDoomBringer committed:
machines only work in starting spots

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	html/changelogs/.all_changelog.yml

Commit:
Merge branch 'master' into toolbox3

# Conflicts:
#	html/changelogs/.all_changelog.yml

MrDoomBringer committed:
Revert "training machine!"

This reverts commit f461e12b204676d3f8cffc9abfdb2b2b943942ca.

MrDoomBringer committed:
e

MrDoomBringer committed:
r

MrDoomBringer committed:
step_y -> pixel_y

ishitbyabullet committed:
Merge pull request #139 from CitrusGender/Toolbox3

merge conflicts etc

MrDoomBringer committed:
Merge remote-tracking branch 'rad/2020-Fall-Festival' into trainingmachines

ishitbyabullet committed:
Merge pull request #129 from dragomagol/rollercoaster-fixes

Rollercoaster fixes

MrDoomBringer committed:
Merge remote-tracking branch 'rad/2020-Fall-Festival' into trainingmachines

MrDoomBringer committed:
map

ishitbyabullet committed:
Merge pull request #138 from MrDoomBringer/trainingmachines

Trainingmachines

ishitbyabullet committed:
i forgot these

woops

ishitbyabullet committed:
try again

ishitbyabullet committed:
Merge pull request #140 from ishitbyabullet/crunch-unch

The thermal lance is melted

MrDoomBringer committed:
its free!

ishitbyabullet committed:
this way gang

bobbah committed:
add interview + resolve merge conflicts

ishitbyabullet committed:
Merge pull request #141 from MrDoomBringer/tweaks

makes medical kiosks actually be free

bobbah committed:
cherrypick fix

ishitbyabullet committed:
Merge pull request #142 from ishitbyabullet/The-Caretaker

STAGE 6

bobbah committed:
getting there

Commit:
adds outfits for specific teams and adds functionality to pick arena outfits within the robust dress shop

LemonInTheDark committed:
suffering is finally over

zxaber committed:
cameras are done now

ishitbyabullet committed:
Merge pull request #145 from zxaber/eyes_in_the_sky

Arena camera finalizations

ishitbyabullet committed:
Merge pull request #143 from CitrusGender/Toolbox3

Adds arena outfits and robust dress shop functionality

ishitbyabullet committed:
Merge pull request #144 from LemonInTheDark/ball

suffering is finally over

ishitbyabullet committed:
stage 7

what the fuc-

ishitbyabullet committed:
Merge pull request #146 from ishitbyabullet/stage-7

removes the club penguin references

ishitbyabullet committed:
critical fix

ishitbyabullet committed:
Merge pull request #147 from ishitbyabullet/stage-7

The Final PR

bobbah committed:
changes go brr

bobbah committed:
CHANGES GO BRRRR

RaveRadbury committed:
Merge pull request #148 from bobbahbrown/2020-Fall-Festival

[WIP] Port Interview System from Summer Event

LemonInTheDark commented:
The time has come

CitrusGender commented:
donezo, thank you everyone who helped out.
